### DISCUSSION TITLE
Welcome to Mini-SiD Discussions!

### DISCUSSION THREAD

**Discussion by NuclearPhoenixx** [Open-Gamma-Project/Mini-SiD#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dc1rdb** [comment]
Hi, my name is Robert and I built my first geiger counter in the mid 70s. Still remember the Valvo geiger tube (18503 / ZP1200) being really expensive back then. The geiger counter still works, btw.
About a year ago I became hooked on gamma spectrometry through NuclearPhoenix's open gamma project by building my first detector using his SiPM board and a 1" dia x 1" NaI(Tl) scintillator from Ost Photonics. I am currently experimenting with both the Mini SiD and the Open Gamma Detector in various setups and scintillator configurations.
Best wishes

**Reply by NuclearPhoenixx** [reply]
Great to see you here, Robert 🙂

**Comment by buihuy1203** [comment]
Hi, my name is Huy and I'm building this gamma detector based on your design. I want to ask that the SIG pin will generate raw data right, so how to analyze it into a spectrum since I'm a new person to this field, can you tell me more about this, like how to convert the voltage data into counts and keV.
Thank you very much!

**Reply by NuclearPhoenixx** [reply]
Hi there Huy, I think if you're looking for spectrometry this project is better suited for your needs: https://github.com/OpenGammaProject/Open-Gamma-Detector

**Reply by buihuy1203** [reply]
Thank you ! I also want to ask that can I use the SiPM without the mounting plate, I mean I just using it connected by wires, it's that OK ?

**Reply by NuclearPhoenixx** [reply]
Sure you can. It's better to have the PCB since it mounts securely and is easier to handle, as well as the additional decoupling which will help with noise. It should work nonetheless.
